test/functional/test-runtime.js

```diff
@@ -520,31 +517,44 @@ describe('runtime', () => {
       expect(getServiceForDoc(ampdoc, 'service1')).to.be.instanceOf(Service1);
     });
   });
+});
+
+
+describes.realWin('runtime multidoc', {
+  amp: {ampdoc: 'multi'},
+}, env => {
+  let win;
+  let extensions;
+  let extensionsMock;
+  let ampdocServiceMock;
+
+  beforeEach(() => {
+    win = env.win;
+    win.setTimeout = window.setTimeout;
```

Comment:
Unfortunately, the way sinon implements `useFakeTimers`, I can't override timers in the child window. Not sure what issues we expect eventually though. But I'm 100% not a big fan of this. So any suggestions are appreciated.